### ISSUE TITLE
Add application creation rule

### ISSUE THREAD

**Issue by KhrystynaSuliy** [ita-social-projects/Fragments-back-end#116]
**As a**  registered user **I want to** be able to create application only after populating application title **so that** I can find it in a convenient way later on

**Description**
This user story adds a rule that user is not able to create application without populating the title of the initiative

**Preconditions:**
1. Only registered users are able to create applications.
2. The user is able to create as many applications as they want, the number is not limited. 
3. The user is registered and signed in.
4. ‘Zayavka’ application page exists.

### Acceptance Criteria
1. Add rules for creating application in the system:
- user is not able to create an application without populating the field ‘Назва проєкту’;
- the field ‘Назва проєкту’ must be populated with at least 20 characters.
2. When user populates the filed ‘Назва проєкту’ with < 20 characters and selects button ‘Зберегти’ THEN:
- data is not saved to database;
- application ID is not created;
- user stays on the same page with data populated in the field ‘ Назва проєкту’;
- user can see the borders of the field ‘Назва проєкту’ highlighted with red colour;
- user can see a message: ‘Заявку не створено. Щоб створити заявку, заповніть назву проєкту. Мінімальна кількість символів - 20’.

Out of scope:
- when application is not created, user should receive a notification

**Epic link**
E.g.: Epic #112
